finance_manager/src/main/java/com/manager/finance/transaction/model/entity/Transaction.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * @Autor GapSerg
+ * @Create 2022-08-11.08.2022 13:02
+ **/
+
+package com.manager.finance.transaction.model.entity;
+
+import com.manager.finance.wallet.model.entity.Wallet;
+import com.manager.finance.wallet.model.entity.api.DefaultCurrency;
+import com.sandbox.model.TransactionTypeParameter;
+import java.time.LocalDate;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Entity
+@Table(name = "transactions")
+@Data
+@RequiredArgsConstructor
+@SQLDelete(sql = "UPDATE transactions SET tr_is_deleted = true WHERE tr_id=?")
+@Where(clause = "tr_is_deleted=false")
+public class Transaction {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "tr_id", updatable = false)
+    private Long id;
+
+    @Column(name = "tr_type", nullable = false)
+    private TransactionTypeParameter typeOf;
+
+    @Column(name = "tr_amount", nullable = false)
+    private int amount;
```

Comment:
Could you pls explain the meaning of the amount field?

Reply:
What do you mean?